migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,274 @@
+
+This file includes configurations such as the location of the local repository, alternative sources for remote repositories, inputting credentials, among others.
+
+There are two directories for the storage of the settings.xml file:
+
+- Maven Installation Directory: Found at ${Maven.home}/conf/settings.xml, this location reflects the settings applied universally across all Maven projects on the system.
+- User-Specific Configuration: Located at ${user.home}/.m2/settings.xml, this directory allows for the storage of user-specific Maven settings.
+
+Users can tailor their Maven environment to better suit their development needs by using these configurations.
+
+[NOTE]
+====
+The first `settings.xml` is also referred to as `global settings`, the second is referenced as user settings, both are combined
+with users taking precedence.
+====
+
+[TIP] 
+====
+If you need to create user-specific settings from scratch, it's easiest to copy the global settings from your Maven installation to your *_${user.home}/.m2_* directory.
+Maven's default *_settings.xml_* is a template with comments and examples so you can quickly tweak it to match your needs.
+_If you are looking to learn more about how to manage your various Maven settings, please visit the https://maven.apache.org/settings.html[Maven Settings Reference]._
+====
+
+Coming from {PRODUCT} v8 or Red Hat's v7, you might be familiar with product dependencies hosted by https://maven.repository.redhat.com/ga/com/redhat/ba/[Red Hat's Maven repository].
+However this is no longer needed as the product dependencies are now hosted by https://central.sonatype.com/search?q=bamoe[Maven Central].
+There is no need to include an additional repository in `pom.xml` or `settings.xml`.
+
+=== Step 2: Generate a New Maven Project using Apache Maven
+
+Now that we have Maven properly configured, we can move onto the next step, which is to generate a new Maven project:
+
+[start=1]
+. In a terminal window, create a new local folder to for the new Maven project, example: `/Users/developer`, and then naviate to that folder.
+
+. Create the project using Quarkus with the Maven commands below, this will create a Quarkus project called `my-sample-decision-service`, which will be versioned `1.0.0-SNAPSHOT`, including the extensions `kogito-quarkus, dmn, resteasy-reactive-jackson, quarkus-smallrye-openapi, quarkus-smallrye-health`.
+This will create a Quarkus DMN project with the OpenAPI components to get the OpenAPI end points easily with health checks when deploying to Kubernetes:
+
++
+[source,console,subs="+attributes"]
+----
+mvn io.quarkus:quarkus-maven-plugin:2.16.7.Final:create \
+    -DprojectGroupId=com.ibm.bamoe.samples \
+    -DprojectArtifactId=my-sample-decision-service \
+    -DprojectVersion=1.0.0-SNAPSHOT \
+    -DplatformVersion={QUARKUS_VERSION_SHORT} \
```

Comment:
add the following parameters (they are required to work on offline environment):

-DplatformGroupId=io.quarkus.platform \
-DplatformArtifactId=quarkus-bom \